spec.html

```diff
@@ -29912,12 +29912,11 @@ <h1>The Function Constructor</h1>
       </ul>
 
       <emu-clause id="sec-function-p1-p2-pn-body">
-        <h1>Function ( _p1_, _p2_, &hellip; , _pn_, _body_ )</h1>
+        <h1>Function ( ..._args_ )</h1>
```

Comment:
Function has a length of 1, and this syntax would give it a length of 0 (same for all 4)

Reply:
Ah, good point.

Reply:
ah, true. it would still be nice if we could remove those sections by making the signatures `(arg1, ...rest)` or something.

Reply:
Each of the 4 constructors has a section that says the value of the `length` property is 1. E.g. [20.2.2.1 Function.length](https://tc39.es/ecma262/#sec-function.length)

Reply:
ah, true. it would still be nice if we could remove those sections by making the signatures `(arg1, ...rest)` or something.

Reply:
It would be nice, but I don't think it's possible here without making things worse. E.g., with that particular suggestion, we'd then have to:
- explain that passing 0 args is okay (and isn't equivalent to passing `*undefined*` to `arg1`);
- (perhaps) stress that it's the *last* arg that's distinctive, not the first; and
- say something like `If _arg1_ is absent then let _args_ be an empty List; else, let _args_ be the list-concatenation of « _arg1_ » and _rest_.`, or else go back to a more indirect way of defining `_args_`.